.changeset/early-drinks-love.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/components': minor
+---
+
+Removes deprecated theme variables in favor of functional theme variables
```

Comment:
Breaking change here - minor version bump?

Reply:
Yeah, this is a pretty big breaking change because most color-related system props will break. This document is probably where I'd put the migration guide.

Reply:
gotcha, just saw in that issue that you had also mentioned that should go here - I've started working on it and will push that up here once I have something substantial!